pages/topics/client-distribution.mdx

```diff
@@ -0,0 +1,398 @@
+
+| Value           | Description       |
+| --------------- | ----------------- |
+| stable          | Stable build      |
+| ptb ^1^         | Public test build |
+| canary ^1^      | Alpha test build  |
+| development ^2^ | Development build |
+
+^1^ See the [Help Center article](https://support.discord.com/hc/en-us/articles/360035675191-Discord-Testing-Clients) for more information on Discord testing clients.
+
+^2^ The development build follows the [`canary` web release channel](#web-release-channel) and is not recommended for use. It may be unstable or broken at any time.
+
+###### Desktop Platform Type
+
+| Value | Description |
+| ----- | ----------- |
+| win   | Windows     |
+| osx   | macOS       |
+| linux | Linux       |
+
+###### Desktop Architecture Type
+
+| Value | Description  |
+| ----- | ------------ |
+| x86   | 32-bit build |
+
+###### Desktop Executable Format
+
+| Value  | Description                  |
+| ------ | ---------------------------- |
+| deb    | Debian software package file |
+| tar.gz | Compressed archive file      |
+
+### Mobile
+
+Discord offers stable and beta mobile clients for both Android and iOS.
+
+###### Android Release Channel
+
+<Alert type="warn">
+
+Official Android clients employ a minimum version check at startup. If the installed version is below the minimum version, the client will refuse to start and prompt the user to update.
+
+This minimum version check is available at `https://dl.discordapp.net/apps/android/versions.json`. The response contains a single `discord_android_min_version` field, which is a string containing the minimum version.
+
+</Alert>
+
+| Value     | URL                                                       | Description              |
+| --------- | --------------------------------------------------------- | ------------------------ |
+| stable    | https://play.google.com/store/apps/details?id=com.discord | Stable application build |
+| beta ^1^  | https://play.google.com/apps/testing/com.discord          | Beta application build   |
+| alpha ^1^ | https://groups.google.com/g/discord-android-alpha-testers | Alpha application build  |
+
+^1^ See the [Help Center article](https://support.discord.com/hc/en-us/articles/360035675191-Discord-Testing-Clients) for more information on Discord testing clients.
+
+###### iOS Release Channel
+
+| Value     | URL                                                                  | Description                 |
+| --------- | -------------------------------------------------------------------- | --------------------------- |
+| stable    | https://apps.apple.com/us/app/discord-talk-chat-hang-out/id985746746 | Stable application build    |
+| beta ^1^  | https://testflight.apple.com/join/gdE4pRzI                           | Beta application build      |
+| alpha ^1^ | <private\>                                                           | Internal application builds |
```

Comment:
What’s the point of this `<private\>`

Reply:
There's no available link, so I just put `<private>`